be/src/exec/parquet_reader.cpp

```diff
@@ -85,6 +83,28 @@ Status ParquetReaderWrap::_init_parquet_reader() {
         */
         arrow_reader_properties.set_coerce_int96_timestamp_unit(arrow::TimeUnit::MICRO);
 
+        // default batch size is 64K
+        arrow_reader_properties.set_batch_size(config::arrow_read_batch_size);
```

Comment:
why not change the default batch size ?

Reply:
this is arrow default batch size,
the bigger batch size, the more memory it uses.